ReferenceAdapter/src/main/java/com/chartboost/mediation/referenceadapter/sdk/ReferenceFullscreenActivity.kt

```diff
@@ -119,7 +145,7 @@ class ReferenceFullscreenActivity : AppCompatActivity() {
     /**
      * Create and show an interstitial ad.
      */
-    @SuppressLint("ClickableViewAccessibility")
+    @SuppressLint("ClickableViewAccessibility", "SetTextI18n")
```

Comment:
I mean.... we could also just set the accessibility controls xD (not actually important but you're like suppressing it lol)